src/media/css/detail/more-info.styl

```diff
@@ -0,0 +1,40 @@
+/*
+    The info section below the reviews section, featuring detailed app
+    information (category, package size) and content ratings.
+*/
+@import '../lib';
+
+.app-more-metadata {
+    > div {
```

Comment:
We have a thing going on now in site.styl where we're using ```section.main.full > div```s everywhere. So it's pretty widespread and generic. I would like to make a marketplace-element for that though (```mkt-section```).